feat: chained printing method

zchryst opened the pull request:
Hey @tr3v3r - made a bit of progress today towards the chained printing method, looking for any feedback on this draft. I'll be looking at the image tomorrow

tr3v3r commented:
@zchryst Grate job! Thanks a lot! 

Are you planning to add the same feature for Android?

p.s. I'll review today / tomorrow the draft

zchryst commented:
@tr3v3r Sorry, I'm afraid our project is only using iOS and due to time constraints, I won't be able to commit to the android side. 

I'm hoping that this can at least lay the groundwork for someone else to do that part?

zchryst commented:
Alrighty - I've got the image coming through as well, now converting to a proper PR 🥳

tr3v3r commented on src/printing.ts:
```diff
@@ -0,0 +1,264 @@
+   *
+   */
+  _send(value: any) {
+    return EscPosPrinter.printBuffer(value);
```
Could please also add the stub method for Android to not cause a crash for Android users?

tr3v3r replied:
Could you please add Promise wrapper like for main methods (see getPaperWidth at index.ts ) to provide correct completion for printing task?

tr3v3r commented on CONTRIBUTING.md:
```diff
@@ -2,6 +2,12 @@
+
+Ensure that you have installed git-lfs prior to cloning the repository.
+
 ## Development workflow
```
It also would be nice if you could add short docs or examples in the README.md file

tr3v3r commented:
@zchryst Thanks. I've left several comments. Could you please take a look and then we can merge it into the main branch?


I'll provide the same functionality for Android as well on my own.

zchryst commented:
Thanks @tr3v3r - I'll try fix that up towards the end of the week

zchryst commented:
@tr3v3r - apologies for the delay, I've added the stub, updated the docs, and added the promise wrapper - seems to all be working rather well, though I do not have an android device to test with I'm afraid

tr3v3r commented:
merged. I'll make a short check and will release a new version soon

tr3v3r commented:
@zchryst hi! I've just tested your solution regarding printing image and it seems that it doesn't work ;( ( I'm tested on my Epson printer TM-T20II ). For some reason, the base64 image is not processed correctly. 

On the other hand, I'll found another solution to put images into iOS/Android assets folder ( on native side ) and print image from the giving source just using asset name like:

- ios 
   - assets 
        - image.png
        
  ```
  
  .image('image', 200)
  
  ```
  
  So for now I'll keep your method in the code and rename it to `imageFromBase64` to use in chain printing and add another method 'imageFromAssets' to print from source I've mentioned above.
  
  It would be nice if could investigate what the problem and fix if it possible. Thanks
